docs/trace_collection.md

```diff
@@ -92,6 +92,24 @@ class SampleApplication : Application() {
     ```kotlin
     span.setTag("http.url", url)
     ```
+7. Logging a throwable or an error message for a Span:
+
+    Using the Official Opentracing API
```

Comment:
If you want to mark a span as having an error, you can do so by logging it using the official OpenTracing tags :